MAINTAINERS.md

```diff
@@ -77,8 +87,73 @@ Make frequent project releases to the community.
+> 
+> - https://github.com/opensearch-project/OpenSearch/pull/877 — Added a broken link checker
+> - https://github.com/opensearch-project/OpenSearch/issues/894 — Fixed broken links across the OpenSearch-Project
+> - https://github.com/opensearch-project/opensearch-build/pull/17  — Updated labels across all OpenSearch repos
+> - https://github.com/opensearch-project/opensearch-plugins/pull/70 — Updated plugin documentation
+> 
+> Vacha has worked tenaciously on non-glamorous tasks, because they raise the overall Engineering bar for the entire project. 
+
+### Interest
+
+Upon receiving at least 3 positive maintainer votes, and no vetoes, from other maintainers within a maximum period of 2 weeks, the nominating maintainer asks a potential nominee whether they might be interested in becoming a maintainer on the repository, which typically happens via e-mail obtained via DCO, but may also be a comment to a pull request, following a substantial contribution.
+
+> This is great work, Vacha! Would you be interested in becoming a [co-maintainer](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md) on this project? I’d be honored to nominate you.
+
+Individuals accept the nomination by replying, or commenting, for example _"Thank you! I would love to."_
+
+### Public Confirmation
+
+Upon receiving 3 positive maintainer votes, and no vetoes, from other maintainers, and after having confirmed interest with the nominee, the maintainer opens a public issue in order to make the vote visible to the community. 
+
+> _Content from the above nomination._
+> 
+> Here is my +1.
+> 
+> If you support this nomination, please add your +1 or a comment. 
+> 
+> If you do not support this nomination, we invite you to e-mail all/some of the current [project maintainers](https://github.com/opensearch-project/OpenSearch/blob/main/MAINTAINERS.md) in the next 2 weeks. We will keep your feedback private to this group and discuss offline. We will not publicly post the reasons for a decision not to add this co-maintainer, or publicize your name. 
+> 
+> Anyone, including maintainers, can publicly add +1s as requested in the nomination issue if they support the nomination or contact the maintainers in private if they oppose. The issue stays open for 2 weeks.
+
+### Maintainer Decision
+
+If a positive decision has been reached, the nomination is accepted and the nominating maintainer can thus comment on the issue with _"maintainers has been added"_. If a decision cannot be reached for any reason, the nomination is rejected with _"a decision could not be reached"_ with no further explanation necessary.
+
+### Adding the New Maintainer 
+
+The Repo [Admin](ADMINS.md) adjusts the new maintainer’s permissions accordingly. The nominating maintainer opens a pull request to add the new maintainer to MAINTAINERS.md, and closes the nomination issue when that is merged and the individual has been given maintainer-level permissions.
```

Comment:
There is only one listed admin in this repo, is that accurate?

Reply:
I removed the link, it's misleading that it's just in this repo.